docs/user_guide/2_connecting_the_cables.rst

```diff
@@ -0,0 +1,36 @@
+Connecting Cables
+------------------
+
+* Connect the hands to the NUC-CONTROL. It is very important that the exact USB-ethernet adapters are used.
+
+  * The right hand should be connected to a USB->ethernet adapter labelled: ``HAND RIGHT``, which should be connected to one of the USB ports of the NUC-CONTROL (it does not matter which one).
```

Comment:
You start word `ethernet` with capital letter bellow. Maybe here too for consistency?

Reply:
I have replaced in all the file